Examples/UIExplorer/NavigatorIOSColorsExample.js

```diff
@@ -66,6 +66,7 @@ var NavigatorIOSColors = React.createClass({
         tintColor="#FFFFFF"
         barTintColor="#183E63"
         titleTextColor="#FFFFFF"
+        fontFamily="Helvetica"
```

Comment:
I'm pretty certain this affects the font family of the left and right buttons as well, so I don't know if titleFontFamily would be applicable?

I can test this and confirm for you if this will affect more than just the title.